tests/kernel/fatal/message_capture/prj.conf

```diff
@@ -1,2 +1,3 @@
+CONFIG_TEST=y
 CONFIG_LOG=y
 CONFIG_LOG_MODE_MINIMAL=y
```

Comment:
both those lines are selected by `CONFIG_TEST` already, so you can remove them

Reply:
@carlescufi are you sure this is the right thing to do?
config TEST does not select LOG and LOG_MINIMAL, directly.
Instead, those are selected by TEST_LOGGING_DEFAULTS, which is, now, default y, but this might change; maybe it is not good to introduce a dependency that might be hard to track. 

No strong opinion though, just asking.

Reply:
> Instead, those are selected by TEST_LOGGING_DEFAULTS, which is, now, default y, but this might change; maybe it is not good to introduce a dependency that might be hard to track.

why would this change? This has been working like this for some time already. Enabling logging (minimal) for testing is what allows us to see what exactly happens when a test fails, i.e. crashes and fatal exceptions etc.
So I guess CONFIG_TEST alone should be fine and you can remove the logging part since it is implied

Reply:
Nice response, @nashif, agreed. I will remove them.

Reply:
@carlescufi removed